kivy/properties.pyx

```diff
@@ -1692,7 +1692,7 @@ cdef class VariableListProperty(Property):
             Not currently used.
 
     Keeping in mind that the `default` list is expanded to a list of length 4,
-    here are some examples of how VariabelListProperty's are handled.
+    here are some examples of how VariableListProperties are handled.
```

Comment:
```suggestion
    here are some examples of how VariableListProperty is handled.
```

Good catch. But I think the singular avoids the issues you mentioned and seems correct.

Reply:
Agreed -- I think I accepted the suggestion, but it's kinda hard to tell!